### DISCUSSION TITLE
How to activate code

### DISCUSSION THREAD

**Discussion by CrazyMew37** [wavebeem/pkmn.help#183] [Q&A] [answered]
Hello! I'm new to TypeScript, and I just added some new code to a private copy I have of this website. I'm just wondering how I can activate this code (Preferably without a server and offline) and use the website with the new types I added to this.

**Comment by wavebeem** [comment]
Hi! Are you having issues with the instructions from the README? https://github.com/wavebeem/pkmn.help/blob/master/README.md

You will need Node.js installed in order to run the project, then run the commands `npm install` and `npm start`.

Also, I know you asked for no server and offline, but you could definitely run this on a Netlify free account.

**Reply by CrazyMew37** [reply]
Thanks for the info!

**Reply by CrazyMew37** [reply]
Installing Node.JS did a lot of crazy stuff, let's hope that works!

Wow, that's crazy impressive! Thanks for letting me see my new types in action!

**Reply by wavebeem** [reply, marked as answer]
Happy to help!

You may have seen that I'm aware several fans want custom types (https://github.com/wavebeem/pkmn.help/issues/159), but it's quite a lot harder to add in a _general_ case rather than hard coding... so I haven't gotten around to it. If you're motivated enough to add the general feature, I'm happy to answer questions and provide code review along the way.

**Comment by tungchiahui** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
